website/docs/d/is_security_group_rule.html.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_is_security_group_rule"
+description: |-
+  Get information about is_security_group_rule
+subcategory: "Virtual Private Cloud API"
+---
+
+# ibm_is_security_group_rule
+
+Provides a read-only data source for is_security_group_rule. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_is_security_group_rule" "example" {
+	id = ibm_is_security_group_rule.testacc_security_group_rule_all.id
+	security_group_id = ibm_is_security_group.testacc_security_group.id
+}
+
+data "ibm_is_security_group_rule" "example" {
+	id = ibm_is_security_group_rule.testacc_security_group_rule_udp.id
+	security_group_id = ibm_is_security_group.testacc_security_group.id
+}
+
+data "ibm_is_security_group_rule" "example" {
+	id = ibm_is_security_group_rule.testacc_security_group_rule_tcp.id
+	security_group_id = ibm_is_security_group.testacc_security_group.id
+}
+```
+
+## Argument Reference
+
+Review the argument reference that you can specify for your data source.
+
+- `id` - (Required, Forces new resource, String) The rule identifier.
+- `security_group_id` - (Required, Forces new resource, String) The security group identifier.
+
+## Attribute Reference
+
+In addition to all argument references listed, you can access the following attribute references after your data source is created.
+
+- `id` - The unique identifier of the is_security_group_rule.
+- `code` - (Optional, Integer) The ICMP traffic code to allow.
```

Comment:
We usually don't mention Optional/Required in the datasources attributes. Only keeping the Type should be fine

Reply:
done.